docs/use-github-in-bcgov/start-working-in-bcgov-github-organization.md

```diff
@@ -62,6 +62,15 @@ In both cases, the basic steps to release the code are similar, while the implic
 
 If you are intending to maintain an active project, make sure to establish the appropriate processes and terms to manage contributions.
 
+## Integrating GitHub Apps
+
+We have large volumes of asks about adding GitHub Apps to our organization’s repository. To manage the asks, we created a request process through the devops-requests repository.
+If the existing applications, like SonarCloud, does not work for you or your team, please create an issue. We need: 
```

Comment:
we've recently decided not to keep using sonar cloud. @MonicaG is there a better example of one we get requests for commonly?

Reply:
Maybe just take the example out?

Reply:
```suggestion
If the existing applications don't work for you or your team, please create an issue. We need: 
```